serialize/Deserializer.cpp

```diff
@@ -1,13 +1,99 @@
 // Copyright (c) 2014-2017 Josh Blum
+//                    2020 Nicholas Corgan
 // SPDX-License-Identifier: BSL-1.0
 
 #include "SerializeCommon.hpp"
 #include <Pothos/Framework.hpp>
 #include <Poco/ByteOrder.h>
 #include <Poco/Format.h>
+
 #include <sstream>
 #include <cstring>
 #include <cassert>
+#include <numeric>
+
+/*
+ * Utility code
+ */
+
+static inline size_t getTotalBufferSize(const std::vector<Pothos::BufferChunk>& buffers)
+{
+    return std::accumulate(
+               buffers.begin(),
+               buffers.end(),
+               0ULL,
```

Comment:
I guess for the perfectionists, unsigned long long != size_t, so maybe this should be size_t(0)